src/main/java/com/microsoft/aad/msal4j/StringHelper.java

```diff
@@ -27,4 +27,19 @@ static String createBase64EncodedSha256Hash(String stringToHash){
         }
         return base64EncodedSha256Hash;
     }
+
+    static String createSha256Hash(String stringToHash){
```

Comment:
Can `createBase64EncodedSha256Hash` be refactored to use this new method?

Reply:
ok